day2.py

```diff
@@ -0,0 +1,32 @@
+
+def helper(vals):
```

Comment:
Suggest naming this something more descriptive.  it doesn't have to be `check_if_ascending_or_descending_and_diffs_are_between_0_and_4`, but maybe something like `is_safe`